src/nvim/normal.c

```diff
@@ -124,26 +120,22 @@ typedef void (*nv_func_T)(cmdarg_T *cap);
 #define NV_KEEPREG  0x100       // don't clear regname
 #define NV_NCW      0x200       // not allowed in command-line window
 
-/*
- * Generally speaking, every Normal mode command should either clear any
- * pending operator (with *clearop*()), or set the motion type variable
- * oap->motion_type.
- *
- * When a cursor motion command is made, it is marked as being a character or
- * line oriented motion.  Then, if an operator is in effect, the operation
- * becomes character or line oriented accordingly.
- */
-
-/*
- * This table contains one entry for every Normal or Visual mode command.
- * The order doesn't matter, init_normal_cmds() will create a sorted index.
- * It is faster when all keys from zero to '~' are present.
- */
+// Generally speaking, every Normal mode command should either clear any
+// pending operator (with *clearop*()), or set the motion type variable
+// oap->motion_type.
+//
+// When a cursor motion command is made, it is marked as being a character or
+// line oriented motion.  Then, if an operator is in effect, the operation
+// becomes character or line oriented accordingly.
+
+/// This table contains one entry for every Normal or Visual mode command.
+/// The order doesn't matter, init_normal_cmds() will create a sorted index.
+/// It is faster when all keys from zero to '~' are present.
 static const struct nv_cmd {
   int cmd_char;                 // (first) command character
   nv_func_T cmd_func;           // function for this command
   uint16_t cmd_flags;           // NV_ flags
-  short cmd_arg;                // value for ca.arg
+  int16_t cmd_arg;                // value for ca.arg
```

Comment:
```suggestion
  int16_t cmd_arg;              ///< value for ca.arg
```
could also adjust the rest to be `///<`